pkg/network/state.go

```diff
@@ -435,7 +435,7 @@ func (ns *networkState) storeClosedConnections(conns []ConnectionStats) {
 				continue
 			}
 
-			if len(client.closedConnections) >= ns.maxClosedConns {
+			if uint(len(client.closedConnections)) >= ns.maxClosedConns {
```

Comment:
Can we support maxClosedConns > maxInt?

Reply:
The type was changed to `uint` because of the default being applied which was already a `uint`.

Reply:
Can we change the default to `int`? There is no way we can support > maxInt entries right?

Reply:
viper doesn't support `uint`, so we are actually casting from `int` to `uint` right away with `uint(cfg.GetInt(join(spNS, "max_closed_connections_buffered")))`. So I don't think it will ever actually be over max int. I can change them both to `int` though.